shell.nix

```diff
@@ -11,9 +12,14 @@ stdenv.mkDerivation {
     export PIP_PREFIX="$(pwd)/_build/pip_packages"
     export PYTHONPATH="$PIP_PREFIX/${pkgs.python3.sitePackages}:$PYTHONPATH"
     export PATH="$PIP_PREFIX/bin:$PATH"
+    # use nix binaries because we don't want to invoke host configs
```

Comment:
On non-NixOS systems, Binaries can be both provided by Nix and host distribution. So we need to make sure that we are using Nix binaries.